Package.resolved

```diff
@@ -23,17 +23,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/groue/GRDB.swift.git",
       "state" : {
-        "revision" : "3df9e8e5c10bd152f02d9aa349a21a0c527e6bda",
-        "version" : "6.2.0"
+        "revision" : "8330469ac3cbbf0ee52b7e8a4e2b1f4c44bb13ea",
+        "version" : "6.4.0"
       }
     },
     {
       "identity" : "ios-sdk",
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/datatrans/ios-sdk.git",
       "state" : {
-        "revision" : "9f799e820372df52fee7587ea9135ceb6485469f",
-        "version" : "2.3.1"
+        "revision" : "600981cf9e4fe421c13105ec9d5ee4b6568c3e97",
+        "version" : "2.4.0"
```

Comment:
Dazu fehlt der Eintrag in der Changelog

Reply:
Der Eintrag steht in der Changelog Version [0.24.0]